docker-compose.yaml

```diff
@@ -0,0 +1,51 @@
+version: '3.7'
+services:
+  keycloak:
+    image: 'jboss/keycloak'
+    ports:
+      - "8080:8080"
+    environment:
+      - KEYCLOAK_USER=admin-test
+      - KEYCLOAK_PASSWORD=admin-test
+      - JDBC_PARAMS='connectTimeout=30'
+      - DB_VENDOR=mariadb
+      - DB_ADDR=mariadb
+      - DB_DATABASE=keycloak
+      - DB_USER=keycloak
+      - DB_PASSWORD=password
+      - JGROUPS_DISCOVERY_PROTOCOL=JDBC_PING
+      - JGROUPS_DISCOVERY_PROPERTIES=datasource_jndi_name=java:jboss/datasources/KeycloakDS,info_writer_sleep_time=500
+    volumes:
+      - ./plugins:/opt/jboss/keycloak/providers
```

Comment:
This didn't work with the `./target:/opt/jboss/keycloak/providers` 
to remove the need for copying the .jar file?